Makefile

```
@@ -0,0 +1,21 @@
+.PHONY: version build build\run run stop clean
+# serverest 
+
+
+NAME_IMAGE=serverest
+PORT=3000
+
+
+build:
+	@DOCKER_BUILDKIT=1 docker build -t ${NAME_IMAGE}/${NAME_IMAGE} .
+
+build/run: build run	
+
+run:
+	@docker run -p  ${PORT}:3000 -d ${NAME_IMAGE}/${NAME_IMAGE}
```

Está sendo enviado o comando `-d para detachar. Dessa forma o dev não irá conseguir ver os dados das requests.
```suggestion
	@docker run -p  ${PORT}:3000 ${NAME_IMAGE}/${NAME_IMAGE}
```

---

ok, feito!